src/adapters/openlayers.adapter.ts

```diff
@@ -44,6 +52,9 @@ export class TerraDrawOpenLayersAdapter extends TerraDrawBaseAdapter {
 		this._lib = config.lib;
 
 		this._geoJSONReader = new this._lib.GeoJSON();
+		this._projection = this._lib.getUserProjection() || "EPSG:4326";
```

Comment:
I had to change this line to `this._projection = this._map.getView().getProjection().getCode() || "EPSG:3857";` and it worked for me.
`

Reply:
cf8313e resolve this. `getUserProjection()` return null if user do not intentionally set projection. And openlayers use `EPSG:3857` in transform methods by default.

Your code actually get **view projection**, which is `EPSG:3857` by default, but not related to APIs terra-draw use. 

`getUserProjection()` gets inner property `userProjection` in module `ol/proj`.

It affects **ALL** API methods[^1] including `getCoordinateFromPixel` and `getPixelFromCoordinate`, which are used by `project()` and `unproject` in openlayers adapter. And that's why user should also inject `toLonLat`, `fromLonLat` for getting correct coordinates with projection other than `EPSG:3857`


[^1]: https://github.com/openlayers/openlayers/blob/v10.1.0/src/ol/proj.js#L559